README.md

```diff
@@ -266,6 +266,20 @@ OPTIONS:
         --ignore <pattern>    Ignore the crates matched by glob
 ```
 
+### Remove dev-dependencies
+
+Remove dev-dependencies from all Cargo.toml files in the workspace. If you dev-dependencies that are
```

Comment:
```suggestion
Remove dev-dependencies from all Cargo.toml files in the workspace. If you have dev-dependencies that are
```

Reply:
Instead of adding this command, can we add an option to `publish` command that does this and resets things back? WDYT? We can make it even the default behavior.

Reply:
Sounds good to me, done